packages/app/universal-ts-utils/src/array/areStringArraysEqual.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns true if the two arrays are equal, false otherwise.
+ * The arrays are considered equal if they have the same length, same order and the elements at each index are equal.
+ */
+export function areStringArraysEqual(a: string[], b: string[]): boolean {
```

Comment:
can you also add documentation for this method to readme.md?

Reply:
Sure! Done in https://github.com/lokalise/shared-ts-libs/pull/280/commits/2698bf73566ae3913fa349bd75c9285c8a55694c